nodejs-object-assign/nodejs-object-assign.spec

```diff
@@ -0,0 +1,50 @@
+%global npm_name object-assign
+%global enable_tests 0
+
+%{?nodejs_find_provides_and_requires}
+
+Name: nodejs-%{npm_name}
+Version: 4.1.1
+Release: 1%{?dist}
+Summary: ES2015 `Object
+License: MIT
+URL: https://github.com/sindresorhus/object-assign
+Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+BuildRequires: nodejs-packaging
+BuildArch:  noarch
+ExclusiveArch: %{nodejs_arches} noarch
+
+%{?nodejs_find_provides_and_requires}
+
+%description
+%{summary}
+
+%prep
+%setup -q -n package
+
+%build
+$BUILD
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr index.js license package.json readme.md %{buildroot}%{nodejs_sitelib}/%{npm_name}
```

Comment:
I just noticed this: since license and readme.md are also in %files they end up twice in the RPM. Not a big deal but should we fix this? I can submit a PR to ```npm2rpm``` to do so.